Updated README

paulinearcita opened the pull request:
Please review this and feel free to give any input on what to add/revise/remove in this file.

paulinearcita committed:
Updated README

Please review this and feel free to give any input on what to add/revise/remove in this file.

aminahaq305 reviewed (commented):
This looks good to me, once we have the design documentation ready we can update this ReadMe to include that. I think one thing we can elaborate on is "where to find traceability from test to implementation". What do you think?

aminahaq305 reviewed (commented):
This looks good to me, once we have the design documentation ready we can update this ReadMe to include that. I think one thing we can elaborate on is "where to find traceability from test to implementation". What do you think?

aminahaq305 committed:
Added more information

I fixed some grammar and added more details pertaining to traceability. Please review.

aminahaq305 commented:
I am merging this pull request because we need to make sure the read me is ready for when Buck grades
